zerver/lib/exceptions.py

```diff
@@ -570,3 +571,15 @@ def __init__(self, msg: str, error_type: str) -> None:
 class ServerNotReadyError(JsonableError):
     code = ErrorCode.SERVER_NOT_READY
     http_status_code = 500
+
+
+class TopicWildcardMentionNotAllowedError(JsonableError):
+    code: ErrorCode = ErrorCode.TOPIC_WILDCARD_MENTION_NOT_ALLOWED
+
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    @override
+    def msg_format() -> str:
+        return _("You do not have permission to use @topic mentions in this topic.")
```

Comment:
We should probably document this as a (very minor) API change, or at least a new example failure to document the error code, since other clients may want to do something with it too.

It would not be a bad idea to split the backend changes as a preparatory PR, since that's ready to merge aside from this documentation issue.

Reply:
> We should probably document this as a (very minor) API change, or at least a new example failure to document the error code, since other clients may want to do something with it too.

Done